tests/onnx/test_onnx_model_export.py

```diff
@@ -424,3 +443,17 @@ def test_model_log_without_specified_conda_env_uses_default_env_with_expected_de
         conda_env = yaml.safe_load(f)
 
     assert conda_env == mlflow.onnx.get_default_conda_env()
+
+
+@pytest.mark.large
+def test_OnnxModelWrapper_predit_handles_list_output(onnx_sklearn_model,  model_path, data):
+    """
+    https://github.com/mlflow/mlflow/issues/2499
+    User encountered issue where an sklearn model, converted to onnx, would return a list response.
+    The issue resulted in an error because MLflow assumed it would be a numpy array. Therefore,
+    the this test validates the service does not receive that error when using such a model.
+    """
+    x, y = data
+    mlflow.onnx.save_model(onnx_sklearn_model, model_path)
+    wrapper = mlflow.onnx._OnnxModelWrapper(model_path + '/model.onnx')
```

Comment:
Rather than instantiating the wrapper in this test directly, we should instead load the model's pyfunc representation via `wrapper = mlflow.pyfunc.load_model(...)`.

Reply:
Done